src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,42 @@
+.nav {
+  &__list {
+    display: grid;
+    grid-template-columns: repeat(8, max-content);
+    margin: 0;
+    padding: 0;
+    height: 100%;
+    list-style-type: none;
+    text-transform: uppercase;
+  }
+
+  &__marker:not(:last-child) {
```

Comment:
Consider using '&:not(:last-child)' instead of '&__marker:not(:last-child)' since it would be more consistent with BEM naming conventions.